src/axom/sidre/core/Group.cpp

```diff
@@ -2292,42 +2293,42 @@ Group* Group::walkPath(std::string& path, bool create_groups_in_path)
 {
   Group* group_ptr = this;
 
-  std::string::size_type pos = detail::find_exclusive(path, s_path_delimiter);
-  if(pos != std::string::npos)
-  {
-    std::vector<std::string> tokens = detail::split(path, s_path_delimiter, pos);
-    std::vector<std::string>::iterator stop = tokens.end() - 1;
+  const std::vector<std::string> path_parts = axom::utilities::Path(path, s_path_delimiter).parts();
```

Comment:
This find'ing the first delimiter then splitting was a tad confusing because the old split was still returning the first token.  Is there something I'm missing? I stepped through the previous code to make sure I was duplicating the prior behavior and it still passes all the tests this way.